delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutesSpec.scala

```diff
@@ -67,28 +74,18 @@ class ResourcesRoutesSpec extends BaseRouteSpec with IOFromMap with CatsIOValues
   private val schema3      = SchemaGen.schema(nxv + "otherSchema", project.value.ref, schemaSource.removeKeys(keywords.id))
   private val tag          = UserTag.unsafe("mytag")
 
-  private val myId  = nxv + "myid"  // Resource created against no schema with id present on the payload
-  private val myId2 = nxv + "myid2" // Resource created against schema1 with id present on the payload
-  private val myId3 = nxv + "myid3" // Resource created against no schema with id passed and present on the payload
-  private val myId4 = nxv + "myid4" // Resource created against schema1 with id passed and present on the payload
-  private val myId5 = nxv + "myid5" // Resource created against schema1 with id passed and present on the payload
-  private val myId6 = nxv + "myid6" // Resource created and tagged against no schema with id present on the payload
-  private val myId7 = nxv + "myid7" // Resource created and tagged against no schema with id present on the payload
-  private val myId8 =
-    nxv + "myid8" // Resource created and tagged against no schema with id passed and present on the payload
```

Comment:
Getting rid of these since the tests are now independent and don't require this prior knowledge